v22.1/row-level-ttl.md

```diff
@@ -0,0 +1,400 @@
+| `ttl_pause` <a name="param-ttl-pause"></a>               | If set, stops the TTL job from executing.                                                                                                                                    | N/A                                 |
+| `ttl_job_cron` <a name="param-ttl-job-cron"></a>         | Frequency at which the TTL job runs, specified using [CRON syntax](https://cron.help).                                                                                       | N/A                                 |
+| `ttl_automatic_column`                                   | If set, use the value of the `crdb_internal_expiration` hidden column. (Currently always set.)                                                                               | N/A                                 |
+
+For more information about TTL-related cluster settings, see [View TTL-related cluster settings](#view-ttl-related-cluster-settings).
+
+## TTL metrics
+
+The table below lists the metrics you can use to monitor the effectiveness of your TTL settings.
+
+| Name                                      | Description                                                      | Measurement          | Type      |
+|-------------------------------------------+------------------------------------------------------------------+----------------------+-----------|
+| `jobs.row_level_ttl.range_total_duration` | Duration for processing a range during row level TTL.            | `nanoseconds`        | Histogram |
+| `jobs.row_level_ttl.select_duration`      | Duration for select requests during row level TTL.               | `nanoseconds`        | Histogram |
+| `jobs.row_level_ttl.delete_duration`      | Duration for delete requests during row level TTL.               | `nanoseconds`        | Histogram |
+| `jobs.row_level_ttl.rows_selected`        | Number of rows selected for deletion by the row level TTL job.   | `num_rows`           | Counter   |
+| `jobs.row_level_ttl.rows_deleted`         | Number of rows deleted by the row level TTL job.                 | `num_rows`           | Counter   |
+| `jobs.row_level_ttl.num_active_ranges`    | Number of active workers attempting to delete for row level TTL. | `num_active_workers` | Count     |
+| `jobs.row_level_ttl.total_rows`           | Approximate number of rows on the TTL table.                     | `total_rows`         | Count     |
+| `jobs.row_level_ttl.total_expired_rows`   | Approximate number of expired rows on the TTL table.             | `total_expired_rows` | Count     |
+
+By default, these metrics are aggregated. If you want to have metrics broken out by table, you must take the following steps:
+
+- Set the `server.child_metrics.enabled` [cluster setting](cluster-settings.html) to `true`.
+- Set the `ttl_label_metrics` storage parameter to `true`. (This step is necessary to reduce the cardinality of the metrics.)
```

Comment:
Just doing the above means all non-ttl_label_metrics'd tables will look like `jobs.row_level_ttl.total_expired_rows{relation_name=default}`. Setting this flag makes it broken out by it's extra table and *adds* cardinality.

(please feel free to give it a shot! it's confusing! you can look at the [`/status/varz`](http://localhost:8081/_status/vars)` endpoint to see how these metric names get reported over these settings.